src/test/java/automationpractice/com/data/Registration.java

```diff
@@ -0,0 +1,20 @@
+package automationpractice.com.data;
+
+import automationpractice.com.domain.Registration;
+
+public class AccountCreation {
+    public static final AccountCreation NEW_ACCOUNT_CREATION = new AccountCreation(
+       name = "firstName",
+            "lastName",
+            "email",
+            "password",
+            "dateOfBirth",
+            "firstNameAddress",
+            "lastNameAddress",
+            "address",
+            "city",
+            "state",
+            "zipCode",
+            "country",
+            "mobilePhone");
+}
```

Comment:
Tutaj nie powinieneś tworzyc obiektu klasy AccountCreation. Jesteś w tej klasie. Dlaczego w nawiasie masz "name = "? To nie jest potrzebne.